crane/cmd.go

```diff
@@ -252,6 +268,11 @@ See the corresponding docker commands for more information.`,
 	cmdLogs.Flags().StringVarP(&options.tail, "tail", "", "all", "Output the specified number of lines at the end of logs")
 	cmdLogs.Flags().BoolVarP(&options.colorize, "colorize", "z", false, "Output the lines with one color per container")
 
+	cmdTmpl.Flags().StringVarP(&options.tmplInput, "template", "t", "", "The input template file name")
```

Comment:
continuing here the discussion about `template` as the long parameter name... I see your point about the fact that it's weird to have both the command and a flag with a similar key... Maybe that calls for a different command name - `generate`, `transform` ?